### PR TITLE
updated doc  mentioning Net Core is now supported

### PR DESCRIPTION
**Pull request by v-shazilms**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/759f1f17739674d6378d47095c039088e5315d7a/msteams-platform%2Fbots%2Fcalls-and-meetings%2Frequirements-considerations-application-hosted-media-bots.md) | 86 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6ca2559e-c9a6-44b6-b88b-d8a668628164) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — DISMISSED** [review]

**Comment by v-shazilms** [comment]
All changes have been completed. Are there extra changes needs to be done if not can it be merge

**Comment by v-shazilms** [comment]
All changes have been completed. Are there extra changes needs to be done if not can it be merge

File: msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ An application-hosted media bot requires the [`Microsoft.Graph.Communications.Ca
 
 An application-hosted media bot requires the following:
 
-* The bot must be developed in C# and the standard .NET Framework and deployed in Microsoft Azure. You can't use C++ or Node.js APIs to access real-time media and .NET Core isn't supported for an application-hosted media bot.
+* The bot must be developed in C# and the standard .NET Framework and deployed in Microsoft Azure. You can't use C++ or Node.js APIs to access real-time media and .NET Core is supported for an application-hosted media bot and sdk supports dotnet 6.
```
Comment (v-ypalikila) [inline msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md:22]: Is donet 6 the correct usage. Should we write it as .NET 6.0 ?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e09d391e7e01046753b0925eef45c51ae5191954/msteams-platform%2Fbots%2Fcalls-and-meetings%2Frequirements-considerations-application-hosted-media-bots.md) | 84 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9d712255-9641-498d-a5bb-7be512a3567e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md (hunk 1 above)
Comment (v-shazilms) [reply msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md]: I have changed it to .Net 6.0.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/be41dce99fe22f7d84f4ffecaae82974b7a90f1e/msteams-platform%2Fbots%2Fcalls-and-meetings%2Frequirements-considerations-application-hosted-media-bots.md) | 85 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/28940e28-4fd9-433d-a385-7c7e03bdd84d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ An application-hosted media bot requires the [`Microsoft.Graph.Communications.Ca
 
 An application-hosted media bot requires the following:
 
-* The bot must be developed in C# and the standard .NET Framework and deployed in Microsoft Azure. You can't use C++ or Node.js APIs to access real-time media and .NET Core isn't supported for an application-hosted media bot.
+* The bot must be developed in C# and the standard .NET Framework and deployed in Microsoft Azure. You can't use C++ or Node.js APIs to access real-time media and .NET Core is supported for an application-hosted media bot and sdk supports .Net 6.0.
```
[the remark this answers is not in the document]
Comment (v-shazilms) [reply msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md]: i have made those changes

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/00a43dc6c4e04310619b53be54cf7eb422dd083f/msteams-platform%2Fbots%2Fcalls-and-meetings%2Frequirements-considerations-application-hosted-media-bots.md) | 86 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dd15854e-dfa9-4707-8708-a70d446691a2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3c1338aadd3835c270676f45f9936712920cdf18/msteams-platform%2Fbots%2Fcalls-and-meetings%2Frequirements-considerations-application-hosted-media-bots.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bff339cb-1c22-4a8a-85fb-1cd6ff0f0cdb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by v-shazilms** [comment]
I have resolved all the conflicts and made all changes, please revive it and merge it , @Wajeed-msft  @v-ypalikila

**Review by v-ypalikila — DISMISSED** [review]
Approved.

**Review by samarthasthana — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3d3fa617934ac3f95646e929d3fe573602af25df/msteams-platform%2Fbots%2Fcalls-and-meetings%2Frequirements-considerations-application-hosted-media-bots.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/52d7bd75-d6ed-4c67-b5c7-0c3e4096a7be) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/706c196a5a6201455c96cba6d8296caf4692edbe/msteams-platform%2Fbots%2Fcalls-and-meetings%2Frequirements-considerations-application-hosted-media-bots.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3adf6d9f-2663-42fd-828b-de73c7616180) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**